akka-actor/src/main/scala-2.13-/akka/util/ByteString.scala

```diff
@@ -138,6 +138,9 @@ object ByteString {
 
   val empty: ByteString = CompactByteString(Array.empty[Byte])
 
+  /** Java API */
+  val emptyByteString: ByteString = empty
```

Comment:
should we mention it in migration guide for 2.6? not that it's a 2.6 change, but some java users will probably use 2.13 at the same time as changing to 2.6 (if we push that version as the recommended for java users)

Reply:
Might make sense, will do.